modules/sensor/src/rgb-depth/realsense/vpRealSense_impl.h

```diff
@@ -391,7 +404,8 @@ void vp_rs_get_pointcloud_impl(const rs::device *m_device, const std::map<rs::st
                 rgb = (static_cast<uint32_t>(color[(i_ * (unsigned int)color_width + j_) * nb_color_pixel]) |
                        static_cast<uint32_t>(color[(i_ * (unsigned int)color_width + j_) * nb_color_pixel + 1]) << 8 |
                        static_cast<uint32_t>(color[(i_ * (unsigned int)color_width + j_) * nb_color_pixel + 2]) << 16);
-              } else {
+              }
+              else {
                 rgb = (static_cast<uint32_t>(color[(i_ * (unsigned int)color_width + j_) * nb_color_pixel]) << 16 |
                        static_cast<uint32_t>(color[(i_ * (unsigned int)color_width + j_) * nb_color_pixel + 1]) << 8 |
                        static_cast<uint32_t>(color[(i_ * (unsigned int)color_width + j_) * nb_color_pixel + 2]));
```

Comment:
Same here if you can modify in the file all the old cast with static_cast for quality rules requirements it would be nice

Reply:
Done in 01950712e